tutorial/branches/simultaneous-changes.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Handling simultaneous changes and conflicts
+permalink: /tutorial/branches/simultaneous-changes/
+---
+
+In the [previous section](/tutorial/branches/project-diff/) you examined the project diff to see what has been changed in the branch. You now know what changes will be applied when the branch is merged. But before you proceed to merging the development branch to production, there is one more scenario that needs to be mentioned. While you were working in your development branch, someone else might have made changes to your production branch in the meantime.
```

Comment:
toto bych úplně vyjmul z tutoriálu a dal pod /components/branches/, to je zase myšlenkovej úkrok stranou
bez kterýho se tutoriál obejde

Reply:
https://keboola.atlassian.net/browse/KBC-1340